.github/workflows/ci-cd.yml

```
@@ -19,34 +20,38 @@ jobs:
           token: ${{ secrets.PRIVATE_REPO_TOKEN }}
           path: config
 
+      # 2. 설정 파일 복사
       - name: Copy config files (application*.yml)
         run: |
           mkdir -p src/main/resources/
           cp config/application-*.yml src/main/resources/
 
+      # 3. JDK 설정
       - name: Set up JDK 17
         uses: actions/setup-java@v3
```

actionlint (1.7.7)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)